test/optimizations/autoLocalAccess/localArrays-1D-COO.good

```diff
@@ -12,7 +12,7 @@ End analyzing forall (localArrays.chpl:56)
 
 Static check successful. Using localAccess (localArrays.chpl:57)
 Testing 1D COO
-2
+1 2
```

Comment:
While the changes look good, I can't understand the good file before your change. It looks like it should have always been printing out `1 2`. Do you have a guess why it wasn't? I don't really remember.